tests/run.sh

```diff
@@ -127,12 +127,12 @@ if [ "${1-}" = '--debug' ]; then
     read line
 fi
 
-# set to the case name you want to run only for debug
-do_case=""
+# List the case names to run, separate them by space, eg. "binlog gencol"
+do_cases=""
```

Comment:
how about use array in shell  https://stackoverflow.com/questions/1878882/arrays-in-unix-shell

Reply:
Is there an easy way to test if a case is among the selected ones with array?

Reply:
if do_cases' length is not 0, just run code like:
```
for case in $do_cases; do
      sh tests/$case/run.sh
done
```